importMariaDBdmpFile/runDBUpdate.py

```diff
@@ -0,0 +1,169 @@
+#!/usr/bin/env python2
+'''
+Retrieve a mysqldump file from a given location on a remote host and use mysql
+to import the database locally. The remote file can be compressed as a .zip
+archive, in which case it is inflated. After successful load, the dump file
+is archived.
+'''
+import os
+import sys
+import shutil
+import logging
+import subprocess
+from time import gmtime, strftime
+import ConfigParser
+import zipfile
+
+
+class Conf:
+    '''
+    Make accessible the parameters, configuration and set up logging
+    '''
+    def __init__(self, path):
+
+        config = ConfigParser.ConfigParser()
+
+        config.read(path)
+
+        self.remoteHost = config.get('remote', 'host')
+        self.remoteUser = config.get('remote', 'user')
+        self.remotePath = config.get('remote', 'path')
+
+        self.workDir = config.get('local', 'workDir')
+        self.archiveDir = config.get('local', 'archiveDir')
+        self.mysqlOptionsPath = config.get('local', 'mysqloptions')
+        self.format = config.get('local', 'format')
+
+        logConfig = config.get('logs', 'level')
+
+        if logConfig == 'info':
+            logLevel = logging.INFO
+        else:
+            logLevel = logging.ERROR
+
+        logging.basicConfig(level=logLevel)
```

Comment:
You might be able to omit the `if` statement.

> The level parameter now accepts a string representation of the level such as ‘INFO’

https://docs.python.org/3/library/logging.html#logging.Logger.setLevel

i.e. (and untested)

```suggestion
        logConfig = config.get('logs', 'level')
        logging.basicConfig(level=logLevel)
```

Reply:
AFAIKS this is not available in Python 2 and I've assumed Python 2 thus far.

Reply:
Ah, you can assume Python 3 (at least Python 3.6.8)